repository/release.go

```diff
@@ -76,7 +75,7 @@ func (r *ReleaseRepository) readRelease(ctx context.Context, q querier, readQuer
 		return svcmodel.Release{}, err
 	}
 
-	tagURL, err := r.generateGitTagURLForRelease(rls)
+	tagURL, err := r.githubURLGenerator.GenerateGitTagURL(rls.GithubOwnerSlug.String, rls.GithubRepoSlug.String, rls.GitTagName)
```

Comment:
When fetching a release, I should always have the GitHub owner and slug. Therefore, I call GenerateGitTagURL directly without checking the values first. If empty values are provided, the GitHub client returns an error.